migrations/20200719025533-add-name-column-users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        "users",
+        "name",
+        {
+          type: Sequelize.DataTypes.STRING,
+        },
+        { transaction: t }
+      )
+      await queryInterface.sequelize.query(
+        `UPDATE users SET "name" = CONCAT("firstName", ' ', "lastName")`,
+        { transaction: t }
+      )
```

Comment:
i added a [migration that adds an admin user](https://github.com/uwblueprint/paramedics-web/blob/master/migrations/20200717012421-load-admin-user.js) before this change, which sets `lastName` to an empty string, which appends a space once this migration is run - this suggestion is to account for that edge case, but test it first cause i dunno if it's quite accurate

```suggestion
      await queryInterface.sequelize.query(
        `UPDATE users SET "name" = "firstName" WHERE "lastName"=''`,
        { transaction: t }
      );
      await queryInterface.sequelize.query(
        `UPDATE users SET "name" = CONCAT("firstName", ' ', "lastName") WHERE "lastName"<>''`,
        { transaction: t }
      );
```